SU-HSDO/suhumsci
# STN-543 : Add styles for buttons within lists on node detail pages

# READY FOR REVIEW

## Summary
This PR adds styles for buttons within a list on node detail pages. 
This includes:
* hs-button
* hs-button--big
* hs-secondary-button

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests pass.
2. Add a list of buttons to a node detail page. I recommend the [person node detail page](http://sparkbox-sandbox.suhumsci.loc/people/qa-person-all-fields-filled)
3. I have added lists of buttons on the [person node detail page on staging](https://sparkbox-sandbox-stage.stanford.edu/people/qa-person-all-fields-filled). You can either sync with staging or follow the steps below:
* On the node detail page, click on `Manage display` in the blue Drupal admin menu
* Click on `Manage layout`
* Within Section 1, click on `+ Add block` in the 3rd column (sidebar)
* Under `Content fields` click on `Info Links`
* Add `hs-button` to the `Field Formatter Class
* Click `Add block`
* Save the layout changes and go back to person node detail page to review the list of primary buttons
* Repeat the above steps for `hs-secondary-button` and `hs-button--big`
